include/arch/arm/aarch64/sys_io.h

```diff
@@ -25,7 +25,9 @@ extern "C" {
 
 static ALWAYS_INLINE uint8_t sys_read8(mem_addr_t addr)
```

Comment:
Might be good to add a comment in the source similar to the one in the commit message.